reactioncommerce/reaction-next-starterkit
# feat: Product grid pagination

Issue: #14
Impact: **minor**
Type: **feat**

## Issue

Add pagination to product grid

## Solution

- Update `withCatalogItems` to support pagination
- Create a generic pagination component
- Create a generaic paganation helper in lib
- Add Paganation to product grid component

## Testing

1. load many products into a running copy of reaction using devtools or manually
2. load http://localhost:4000 and use the next / previous buttons to go through pages
3. Hard refresh the window, ensure your results reflect the page you think you're supposed to be on
